### DISCUSSION TITLE
can't install route for 192.168.11.133/32 === 192.168.11.132/32 out, conflicts with IKE traffic 13[IKE] unable to install IPsec policies (SPD) in kernel

### DISCUSSION THREAD

**Discussion by hcjjmt** [strongswan/strongswan#1007] [Q&A]
can't install route for 192.168.11.133/32 === 192.168.11.132/32 out, conflicts with IKE traffic
13[IKE] unable to install IPsec policies (SPD) in kernel
![image](https://user-images.githubusercontent.com/34616626/164878741-bd27c0fe-6e39-4be3-b31c-14f5bf1c87e4.png)

**Comment by tobiasbrunner** [comment]
Sounds like you are trying to negotiate a host-to-host connection with the _kernel-libipsec_ plugin enabled. That only works with [special config options](https://docs.strongswan.org/strongswan-docs/5.9/plugins/kernel-libipsec.html#_host_to_host_tunnels). To avoid it, use the kernel's IPsec implementation via _kernel-netlink_ plugin.

**Comment by bin-wang1** [comment]
咋样了，解决了吗？
